packages/core/src/components/dropdown/index.js

```diff
@@ -26,9 +26,9 @@ function tryAndPass(cond, str) {
 }
 
 function htmlToElement(html) {
-  const template = document.createElement('template');
```

Comment:
why div instead of template?

Reply:
Unfortunately IE11 does not support `template`. And in `www-website` no proper polyfill provided which leading to error. Which described in this ticket https://jira.weworkers.io/browse/GROW-10169
This is solution. To our purposes it's doesn't matter what element to use for `string->dom` conversion since it does not attached to DOM tree and `"deleted"` by garbage collector after use. But you're right `template` is correct option. 
Btw as `ray` is free-to-use tool we should try to make it less dependent on other tools such as polyfills e.t.c